cursive-core/src/printer.rs

```diff
@@ -18,10 +19,9 @@ use unicode_width::UnicodeWidthStr;
 
 /// Convenient interface to draw on a subset of the screen.
 ///
-/// The area it can print on is defined by `offset` and `size`.
-///
-/// The part of the content it will print is defined by `content_offset`
-/// and `size`.
+/// The area it can print on is defined by `offset` and `size`.\
```

Comment:
Why this change?

[The doc format](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html) (much like the git message format) relies on a single short line, followed by a blank line, followed by the rest of the documentation.

Reply:
Using less words to say the same thing improve the readability of the sentence. Here the goal is not to loose the reader using few unnecessary words.

Furthermore, you have the short line (line 20), and then the rest of the documentation (line 22 to line 24).

The goal of the blank line is to create few paragraphs. If you use the "\" character, then you add a carriage return *inside* your paragraph. So you need to use the blank line if you want to create another paragraph, and use the "\" character if you want to structure a single paragraph. That's why I didn't add the "\" char at the end of each line.

Reply:
Ah indeed, I thought this was the first line of the doc comment.

Since it's just to merge the two paragraphs into a single one with newline?

Reply:
Yes, that's it. It makes the doc more easier to read than having a new paragraph for each sentence. :smile: